### PR TITLE
 Add dogstatsd support

### PR DESCRIPTION
**Pull request by cep21**
dogstatsd support

### CODE REVIEW

**Comment by cep21** [comment]
@tedoc2000

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,22 +6,27 @@ subsections are relevant.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.0.9] - 2015-10-26
```
Comment (mdubbyap) [inline CHANGELOG.md]: no word about dogstatsd?

File: requirements.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 psutil==3.2.2
+simplejson==2.1.0
```
Comment (mdubbyap) [inline requirements.txt]: simplejson got taken out, please remove

**Comment by mdubbyap** [comment]
Please add the port to the config file like @Pquincyl  documented:
https://docs.google.com/document/d/1H7kTyZ7OvjVVAEafHbHIwBKZAiLkP4qZR5fv2n35hYo/edit?ts=56340efc#

Also update the README.md with the new functionality.

**Comment by mdubbyap** [comment]
Why do only some of the python files have flake8 or pylint run on them?

**Comment by cep21** [comment]
datadog's files fail flake8 and pylint spectacularly so I don't run it on those

**Comment by mdubbyap** [comment]
Can you fix the .travis.yml then to avoid those files.  Thanks.

**Comment by cep21** [comment]
https://github.com/DataDog/dd-agent

```
Required:

python 2.6 or 2.7
```

**Comment by cep21** [comment]
@mdubbyap @tedoc2000

**Comment by mdubbyap** [comment]
Still waiting on changes to the README

**Pull request merged**